atom/browser/login_handler.h

```diff
@@ -40,7 +40,7 @@ class LoginHandler : public base::RefCountedThreadSafe<LoginHandler> {
   // thread.
   content::WebContents* GetWebContents() const;
 
-  const net::AuthChallengeInfo* auth_info() const { return &auth_info_; }
+  const net::AuthChallengeInfo* auth_info() const { return &*auth_info_; }
```

Comment:
This can return a `scoped_refptr` instead.

Reply:
Additional copy for return would mean incrementing and decrementing the refcount.
I'd prefer to keep this as is, unless you see a significant advantage of doing that.

Reply:
The current usage of this getter is for an event emitted on the UI thread, if the login handler gets destroyed before the event it would not crash when the auth info is ref counted.